features/admin.core.v1/components/header.tsx

```diff
@@ -125,6 +126,11 @@ const Header: FunctionComponent<HeaderPropsInterface> = ({
     const { isOrganizationManagementEnabled } = useGlobalVariables();
     const eventPublisher: EventPublisher = EventPublisher.getInstance();
 
+    const {
+        data: myAccountApplicationData,
+        isLoading: isMyAccountApplicationDataFetchRequestLoading
+    } = useMyAccountApplicationData();
```

Comment:
Shall we fetch the myaccount application only when showAppSwitchButton is set to true?